File: PRAKTIKUM_3/PRAK304-2410817210004-Rahmad.py
angka = int(input("Masukkan angka: "))
if angka == 0:
    print( f"{angka} adalaha nol")

elif angka < 10 and angka >0:
    print( f"{angka} adalah satuan")

elif angka>10 and angka < 20:
    print( f"{angka} adalah belasan")

elif angka==10 or angka>20 and angka < 100:
    print( f"{angka} adalah puluhan")

else:
    print( f"{angka} adalah ratusan )atau lebih")

File: PRAKTIKUM_3/PRAK304-2410817210004-Rahmad.c
#include <stdio.h>

int main() {
    int a;
    
    printf("Masukkan bilangan : ");
    scanf("%d", &a);
    if(a==0){
    printf("NOL");
    }else if(a>=1&&a<=9){
    printf("satuan");
    }else if(a>=11&&a<=19){
    printf("Belasan");
    }else if(a>=20&&a<=99){
    printf("Puluhan");
    }else{
    printf("Anda menginput melebihi batas limit bilangan");
    }
}
